jacodb-panda-dynamic/src/main/kotlin/org/jacodb/panda/dynamic/ets/dto/convert.kt

```diff
@@ -0,0 +1,712 @@
+/*
```

Comment:
Please, name files starting with a capital letter

Reply:
All files that contain classes are spelled in TitleCase. All files that only contain utility functions, or even just one function, better be spelled the same as functions are spelled - in camelCase, don't you think so?

Feel free to ping me once again and I am going to change it for you. Or feel free to rename them yourself :)